drivers/wifi/nrf700x/CMakeLists.txt

```diff
@@ -107,3 +107,13 @@ zephyr_library_sources_ifdef(CONFIG_NRF700X_ON_SPI
 zephyr_library_sources_ifdef(CONFIG_NRF700X_UTIL
 	zephyr/src/zephyr_wifi_util.c
 )
+
+# RPU FW header file is included in the zephyr_fmac_main.c
+# Run ext_code from the external flash (XIP). No need to copy.
+zephyr_code_relocate(FILES zephyr/src/zephyr_fmac_main.c LOCATION EXTFLASH_RODATA NOCOPY)
```

Comment:
What happens if this is run on nRF7002 DK and the code is relocated, but the linker script does not put the sections in the external flash? Will they be put normally in the internal flash?